main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"PB173-discord-bot/bot"
+	"github.com/joho/godotenv"
+	"github.com/rs/zerolog/log"
+)
+
+func main() {
+	err := godotenv.Load(".env")
+	if err != nil {
+		log.Error().Msg("Couldn't load environment.")
+	} else {
+		log.Info().Msg("Environment var loading successful.")
+	}
```

Comment:
One thing to note, when you would want, for example, to containerize the app a .env file would have to be present in the container otherwise running the app would fail. 
But you do not want to put your .env file into the container so you will have to pass it via environmental variables and put a blank .env file in the container so it is found by this `godotenv.Load(".env")` function call.
Sometimes not finding the `.env` file is completely valid.